### ISSUE TITLE
Path Redundancy (and Diverse)

### ISSUE THREAD

**Issue by carmine** [telstra/open-kilda#1861]
Path Redundancy leverages Path Diversity (anti-affinity) to establish 2 distinct paths between ingress/egress. There is an option as to whether it is active-active or active-passive.

Active-passive means that the alternate path is reserved (bandwidth) and most of the path should be programmed (ie everything except ingress in both directions). The benefit here is only 1 switch will need to be programmed to activate the passive flow.

Blocked by #1231

**Comment by pzakatov** [comment]
looks duplicate of #1232

**Comment by carmine** [comment]
duplicate of #1232

**Comment by pzakatov** [comment]
done
